atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/DbSharedConfig.java

```diff
@@ -26,6 +26,11 @@
 public abstract class DbSharedConfig {
 
     @Value.Default
+    public String tablePrefix() {
+        return "a_";
```

Comment:
Table names have a maximum length (32 characters on oracle?) I didn't see a need to eat up more of those, but I'm happy to make this more verbose.